README.md

```diff
@@ -43,11 +43,13 @@ bazel build -c opt //:nighthawk
 ```
 USAGE:
 
-bazel-bin/nighthawk_client  [--open-loop] [--failure-predicate <<string,
-uint64_t>>] ... [--termination-predicate
-<<string, uint64_t>>] ... [--trace <uri
-format>] [--sequencer-idle-strategy <spin
-|poll|sleep>] [--max-requests-per-connection
+bazel-bin/nighthawk_client  [--open-loop]
+[--h1-connection-reuse-strategy <HOT|FAIR>]
```

Comment:
Are we missing some other parameters like:
- Keep at least N open connections.
- Reuse connections that have been idle for at least N msec.

I think that the least-recently-used strategy results in connections not idling out after a burst.  It also helps spread load across envoy worker threads, but only if you manage to create enough connections in the first place.  Hence the parameter to create at least N connections to get better steady state behavior.

Reply:
- We have `--prefetch-connections` which is an all or nothing proposition. It will switch to proactive connection prefetching mode and keep open the number of connections specified. An enhancement could then be to let that take an int to specify the minimum amount?
- Yeah, those are nice, I also mulled over adding "random/bursty" as a nastier pattern. 

Shall I file issues for these?

Reply:
sgtm.